modules/iam/src/main/scala/ch/epfl/bluebrain/nexus/commons/iam/io/serialization/JsonLdSerialization.scala

```diff
@@ -20,8 +20,18 @@ trait ConfigInstance {
         case other => other
       })
 }
+trait IdentityJsonLdDecoder extends ConfigInstance {
 
-trait EventJsonLdDecoder extends ConfigInstance {
+  /**
+    * Identity decoder which converts JSON-LD representation to ''Identity''
+    */
+  def identityDecoder: Decoder[Identity] = {
+    import io.circe.generic.extras.semiauto._
```

Comment:
Maybe remove the top-level import since the whole package is imported within both the decoder and encoder.